PhpcsChanged/GitWorkflow.php

```diff
@@ -31,7 +32,24 @@ function getGitUnifiedDiff(string $gitFile, string $git, callable $executeComman
 	return $unifiedDiff;
 }
 
-function isNewGitFile(string $gitFile, string $git, callable $executeCommand, callable $debug): bool {
+function isNewGitFile(string $gitFile, string $git, callable $executeCommand, array $options, callable $debug): bool {
+	if ( isset($options['git-branch']) && ! empty($options['git-branch']) ) {
+		return isNewGitFileRemote( $gitFile, $git, $executeCommand, $options, $debug );
+	} else {
+		return isNewGitFileLocal( $gitFile, $git, $executeCommand, $options, $debug );
+	}
+}
+
+function isNewGitFileRemote(string $gitFile, string $git, callable $executeCommand, array $options, callable $debug): bool {
+	$gitStatusCommand = "${git} cat-file -e " . escapeshellarg($options['git-branch']) . ':' . escapeshellarg($gitFile);
+	$debug('checking status of file with command:', $gitStatusCommand);
+	$gitStatusOutput = $executeCommand($gitStatusCommand, $gitStatusOutput = array(), $return_val = 1);
```

Comment:
This function call pattern with inline assignment expressions is unfamiliar to me; is the intent to provide named arguments?